src/group_theory/free_abelian_group.lean

```diff
@@ -32,10 +32,8 @@ Here we use the following variables: `(α β : Type*) (A : Type*) [add_comm_grou
 group it is `α →₀ ℤ`, the functions from `α` to `ℤ` such that all but finitely
 many elements get mapped to zero, however this is not how it is implemented.
 
-* `lift (f : α → A) : free_abelian_group α →+ A` : the group homomorphism induced
-by the map `f`.
-
-* `hom_equiv : (free_abelian_group α →+ A) ≃ (α → A)` : the bijection witnessing adjointness.
+* `lift : (α → A) ≃ free_abelian_group α →+ A` : the group homomorphism induced
+  by the map `α → A`; the bijection witnessing adjointness.
```

Comment:
In the interest of keeping it simple,
```suggestion
* `lift f : free_abelian_group α →+ A` : the group homomorphism induced
  by the map `f : α → A`.
```

Reply:
Sounds good, maybe good to keep it simple.